test/transcoding/llvm.memset.ll

```diff
@@ -17,34 +6,46 @@
 ; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
 
 ; CHECK-SPIRV: TypeInt [[Int8:[0-9]+]] 8 0
-; CHECK-SPIRV: Constant {{[0-9]+}} [[Len:[0-9]+]] 12
+; CHECK-SPIRV: Constant {{[0-9]+}} [[Lenmemset21:[0-9]+]] 4
```

Comment:
This is probably out of scope of this PR, but I wonder why do we have `OpConstant` here instead of `OpConstantComposite`?

Reply:
If I remember correctly, if the length is large enough(like 128), we have to declare a structure with large number of members, while for the array we just have to specify the number of elements.

Reply:
This is the "length" argument which is a scalar, i.e. an `OpConstant`.